asteroid_filterbanks/pcen.py

```diff
@@ -0,0 +1,162 @@
+from torch import nn
+import torch
+from . import transforms
+from typing import Union
+
+try:
+    from typing import TypedDict
+except ImportError:  # Fallback for >= Python 3.7
```

Comment:
@mpariente I added this to support Python 3.7 and older. I'd like to include keep supporting Python 3.7 as it is what Google Colab's kernel currently uses and it's unclear when they will move to 3.8. 

What are your thoughts?
If it's OK, I think we may need to include `typing_extensions` in the `setup.py` as an explicit dependency